config/routes.rb

```diff
@@ -0,0 +1,70 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+  root 'products#index'
+  resources :users, :products, :orders, :order_items, :reviews
```

Comment:
Yeah, we just did that at the beginning so we could start working and plan to prune after we get the controllers all mapped out.